store/oracles.js

```diff
@@ -1,25 +1,31 @@
-import Vue from 'vue'
+import { loadNextPage } from './utils'
 
 export const state = () => ({
-  oracles: {}
+  oracles: [],
+  nextPageUrl: ''
 })
 
 export const mutations = {
   setOracles (state, oracles) {
-    for (let oracle of oracles) {
-      Vue.set(state.oracles, oracle.oracle, oracle)
-    }
+    state.oracles = [...state.oracles, ...oracles.data]
+    state.nextPageUrl = oracles.next
   }
 }
 
 export const actions = {
-  getOracles: async function ({ rootGetters: { middleware }, commit }, { page, limit }) {
+  getLatestOracles: async function ({ rootGetters: { middleware }, commit }) {
     try {
-      const oracles = await middleware.getActiveOracles({ page, limit })
-      commit('setOracles', oracles.data)
+      if (state.nextPageUrl) return
+      const oracles = await middleware.getActiveOracles()
+      commit('setOracles', oracles)
     } catch (e) {
       console.log(e)
       commit('catchError', 'Error', { root: true })
     }
+  },
+  getMoreOracles: async function ({ state: { nextPageUrl }, commit }) {
```

Comment:
```suggestion
  getMore: async function ({ state: { nextPageUrl }, commit }) {
```
I think the entity name is not necessary because all of this stuff in the context of a named vuex module